src/sparseml/pytorch/sparsification/distillation/modifier_per_layer.py

```diff
@@ -40,6 +40,8 @@
 
 _DISTILLATION_TYPES = [torch.nn.Conv2d, torch.nn.Linear]
 
+_DISTILL_PARAM_GROUP_KEY = "distillation_projection_params"
```

Comment:
let's rename to `_DISTILL_PARAM_GROUP_KEY` and add to `__all__`

Reply:
Don't wanna add a "protected" attribute to `__all__` will remove leaading underscore if this needs to be added

Reply:
yes that was the intention, i forgot to remove the `_` after copy paste 🤦